examples/clojure/Earthfile

```diff
@@ -1,17 +1,17 @@
-VERSION 0.8 
+VERSION 0.8
 FROM clojure:temurin-8-lein
 WORKDIR /var/app
 
 build:
-    RUN apt update && apt install zip -y 
+    RUN apt update && apt install zip -y
     COPY earthly-example/ .
     RUN lein uberjar
     RUN ./extractJarVersion target/uberjar/earthly-example-*-standalone.jar > version
     SAVE ARTIFACT target/uberjar/earthly-example-*-standalone.jar earthly-example-standalone.jar
     SAVE ARTIFACT version
 
 docker:
-    FROM openjdk:22-slim-bullseye
+    FROM amazoncorretto:25-alpine3.22
```

Comment:
Why mazoncorretto:**25**-alpine3.22? Everywhere else you use mazoncorretto:**24**-alpine3.22.